README.md

```diff
@@ -1,39 +1,89 @@
 <a href="https://nimblebox.ai/" target="_blank"><img src="./assets/built_at_nbx.svg" align="right"></a>
+[![PyPI - Python
+Version](https://img.shields.io/badge/python-3.6%20%7C%203.7%20%7C%203.8%20%7C%203.9-blue)](https://pypi.org/project/nbox/)
+[![Downloads](https://pepy.tech/badge/nbox)](https://pepy.tech/project/nbox)
+![GitHub](https://img.shields.io/badge/license-Apache--2.0-blueviolet)
 
-# 🏖️ Nbox
+## 🧐 What is Nbox?
 
-`nbox` is NimbleBox.ai's official SDK.
+`nbox` provides first class API support for all NimbleBox.ai infrastructure (NBX-Build, Jobs, Deploy) and services (NBX-Workspaces) components. Write jobs using `nbox.Operators`
 
-> The entire purpose of this package is to make using ML 🥶.
+# 🤷Why nimblebox
 
-```
-pip install nbox
-```
+- Write and execute code in Python
+- Document your code that supports mathematical equations
+- Create/Upload/Share notebooks
+- Import notebooks from your local machine
+- Import/Publish notebooks from/to GitHub
+- Import external datasets (e.g. from Kaggle)
+- Integrate PyTorch, TensorFlow, Keras, OpenCV
+- Share your projects
+- Collaborate with your team
 
-## 🔥 Usage
+# 🚀 Startup Program
 
-`nbox` provides first class support API for all NimbleBox.ai infrastructure (NBX-Build, Jobs, Deploy) and services (NBX-Workspaces) components. Write jobs using `nbox.Operators`:
+<a href="https://nimblebox.ai/startup-program"><img width="1281" alt="image" src="https://user-images.githubusercontent.com/89596037/188064820-c372f895-fef1-4a84-bd95-8a9a5c3d13d1.png">
 
-```python
-from nbox import Operator
-from nbox.nbxlib.ops import Magic
 
-# define a class object
-weekly_trainer: Operator = Magic()
+#### If you're a new startup with(<$1M raised,<3 years since founded) then you're in luck to be the part of our startup program!
+#### Get *$420k* worth of deals on your favorite tools 
+##### <a href="https://nimblebox.ai/startup-program">Check it out !
+# 🎚 Features
 
-# call your operators
-weekly_trainer(
-  pass_values = "directly",
-)
+### 🏗️ Freedom To Build
+![Build Landing Page (2)](https://user-images.githubusercontent.com/89596037/181773716-ba63f167-af0d-48aa-921a-02e13238c0f2.gif)
 
-# confident? deploy it to your cloud
-weekly_trainer.deploy(
-  job_id_or_name = "magic_jobs",
-  schedule = Schedule(4, 30, ['fri']) # schedule like humans
-)
+
+### 🦾 Automate with Ease
+![Jobs Landing Page](https://user-images.githubusercontent.com/89596037/181774553-99120354-72f5-4064-9216-4f8a5aa050be.gif)
+
+
+
+### 🚀 Intuitive Dashboard
+![Deploy Landing Page](https://user-images.githubusercontent.com/89596037/181775468-cc342a30-d87e-4576-8bdd-8ffdd75ff759.gif)
+
+
+# 🏁 Get Started
+
+
+**Install the package from pypi:**
+
+```pip install nbox```
+
+
+For convinience you should add nbox to your path by setting up an alias. Throughout the rest of the documentation we will be using nbx as the CLI:
+
+```# go to your .bashrc or .zshrc and add
+alias nbx="python3 -m nimblebox "
 ```
```

Comment:
Clarification: newer nbox (>= 0.9.28) add `nbx`, `nbox`, `nimblebox` as scripts– which means they are added as executables in the current (either venv or system python) environment's PATH, so setting the alias is no longer needed 😅